sp800-63b/sec8_security.md

```diff
@@ -39,10 +42,12 @@ This document assumes that the subscriber is not colluding with the attacker who
 | | | A memorized secret is revealed by the subscriber at a bogus verifier website reached through DNS spoofing.
 | Social engineering | The attacker establishes a level of trust with a subscriber in order to convince the subscriber to reveal his or her authenticator secret or authenticator output. | A memorized secret is revealed by the subscriber to an officemate asking for the password on behalf of the subscriber’s boss. |
 | | | A memorized secret is revealed by a subscriber in a telephone inquiry from an attacker masquerading as a system administrator. |
+| | | A One-Time Password sent via SMS is received by an attacker who has convinced the mobile operator to redirect the victims mobile phone to the attacker. |
 | Online guessing | The attacker connects to the verifier online and attempts to guess a valid authenticator output in the context of that verifier. | Online dictionary attacks are used to guess memorized secrets. |
 | | | Online guessing is used to guess authenticator outputs for a one-time password device registered to a legitimate claimant. |
 | Endpoint compromise | Malicious code on the endpoint proxies remote access to a connected authenticator without user consent. | A cryptographic authenticator connected to the endpoint is used to authenticate remote attackers. |
 | | Malicious code on the endpoint causes authentication to other than the intended verifier. | Authentication is performed on behalf of an attacker rather than the subscriber.
+| | | A malicious app on the endpoint relays an SMS One-Time Password to an attacker, who uses it to authenticate.
```

Comment:
As at line 36, "an SMS One-Time Password" -> "an out of band secret received via SMS"